packages/neutron-sdk/src/interchain_queries/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod helpers;
+pub mod hex;
```

Comment:
good question, i wasn't sure that it's useful for everybody.
I'll see if it used elsewhere and make it private it if not

Reply:
it's not easy to remove since used in tests outside of the sdk package. I will merge it with helpers for now. Can discuss for better ways to handle this.

Reply:
yeah, there is a better way, but it's out of scope. will create a task